packages/editor/src/components/post-actions/actions.js

```diff
@@ -535,6 +551,14 @@ const restorePostAction = {
 	},
 };
 
+function useRestorePostAction( postType ) {
+	return useCanUserEligibilityCheckPostType(
+		postType,
+		'update',
+		restorePostAction
```

Comment:
Well, turns out minimising the diff was useful for reusability too 😉

Reply:
Yes for sure the diff change made things easier to reuse.